runtime/src/drivers.rs

```diff
@@ -424,10 +459,11 @@ impl Drivers {
                 .map_err(|_| CaliptraError::RUNTIME_ADD_VALID_PAUSER_MEASUREMENT_TO_DPE_FAILED)?,
             flags: DeriveContextFlags::MAKE_DEFAULT
                 | DeriveContextFlags::CHANGE_LOCALITY
-                | DeriveContextFlags::INPUT_ALLOW_CA
+                | DeriveContextFlags::ALLOW_NEW_CONTEXT_TO_EXPORT
```

Comment:
@clundin25 can you confirm whether we want this attribute here and below?

Reply:
Yes 👍

Reply:
Thanks!